src/common/libjob/specutil.c

```diff
@@ -309,22 +307,21 @@ int specutil_attr_check (json_t *o, char *errbuf, int errbufsz)
     json_t *value;
     const char *errtxt;
 
-    json_object_foreach (o, key, value) {
+    json_object_foreach (o, key, value)
+    {
```

Comment:
ok, I'll ask that you just revert the whitespace changes please.  If any are justified they should go in a separate whitespace commit.

Reply:
Oh, sorry, I don't think I make these changes by hand, I've just been using clang-format on everything. I will revert them.

Reply:
I should have mentioned [rfc 7](https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_7.html) in comments about style.  I believe it refers to our  `.clang-format` at the top level of the source tree which should not have made these changes.  If you were using that then we might have a problem there.

Reply:
I double-checked that I am using clang-format correctly, but it's still making these changes. ``clang-format -style=file -i src/common/libjob/specutil.c``